app/backend/approaches/chatreadretrieveread.py

```diff
@@ -220,7 +222,6 @@ def run(self, history: Sequence[dict[str, str]], overrides: dict[str, Any]) -> A
         # r=self.search_client.search(search_text=None, vectors=[vector], filter="search.ismatch('upload/ospolicydocs/China, climate change and the energy transition.pdf', 'file_name')", top=top)
 
         #  hybrid semantic search using semantic reranker
-       
         if (not self.is_gov_cloud_deployment and overrides.get("semantic_ranker")):
             r = self.search_client.search(
```

Comment:
Missing Async logic here. Add await here.

Reply:
Unfortunately, the sdk doesn't seem to support async and doesn't return a coroutine anywhere.
Won't be able to await this, but also not seeing bottlenecks here currently, we can always wrap in a background task later if we see a need.

Reply:
ahhh ok. I thought I saw it in else statement so suggested change. No worries.